src/components/navigation/search-results.tsx

```diff
@@ -3,11 +3,13 @@ import styled from 'styled-components'
 
 import { inputFontReset } from '../../helper/css'
 
-// TODO: needs type declaration
-type SearchResultProps = any
+interface SearchResultProps {
+  className?: string
+  children: React.ReactNode
+}
 
-export function SearchResults(props: SearchResultProps) {
-  return <SearchResultsWrap {...props}>{props.children}</SearchResultsWrap>
+export function SearchResults({ className, children }: SearchResultProps) {
+  return <SearchResultsWrap className={className}>{children}</SearchResultsWrap>
```

Comment:
What's the point of this React Component if it justs passes the the same props down?

Reply:
just to export messy stying to be honest

Reply:
Mh... but isn't this current thing equivalent to
```tsx
const SearchResults = SearchResultsWrap
```
since `SearchResults` does absolutely nothing (no additional styles, no wrapping, nothing)

Reply:
```export const SearchResults = styled.div```
seems to work, there was a reason for doing it like this at some point… well well.